src/main/java/com/ldtteam/structurize/util/WorldRenderMacros.java

```diff
@@ -976,7 +972,7 @@ public static void renderDebugText(final BlockPos renderPos,
             matrixStack.mulPose(erm.cameraOrientation());
             matrixStack.scale(-0.014f, -0.014f, 0.014f);
 
-            final float backgroundTextOpacity = Minecraft.getInstance().options.getBackgroundOpacity(0.25F);
+            final float backgroundTextOpacity = 0f;
```

Comment:
I'm kinda against this, one just don't render white text on light background

Reply:
the way minecraft renders the background behind the text, just messes up the text. Without it is always better